.github/workflows/publish.yml

```diff
@@ -8,67 +11,83 @@ jobs:
       - uses: linz/action-typescript@9bf69b0f313b3525d3ba3116f26b1aff7eb7a6c0 # v3
         with:
           node-version: 20.x
-      # FIXME: catalog.json is not pushed to the repository (temporary solution)
```

Comment:
Did you intend to remove this comment?

Reply:
Yes, this is intended behavior, the catalog.json should not be commited to git as its a fully generated file.